setup.py

```diff
@@ -9,6 +9,12 @@
 from setuptools import setup
 
 
+def local_scheme(version):
```

Comment:
this one conflicts with the new setup.py

Reply:
Rebasing now.

Reply:
OK, let's try that.

Reply:
i want to revisit this one later i think (the default of the node is to prevent unintended pypi uploads, but i have to think about it for a while)